src/components/Chore.js

```diff
@@ -1,17 +1,32 @@
-import React from "react"
+import React from "react";
+import { Accordion } from "@mui/material";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container"
+import ChoreStep from "./ChoreStep"
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
 
+export default function Chore(props) {
```

Comment:
You can also destructure the props variable in the function definition instead of doing it on line 11. Something like

```
export default function Chore({chore: {name, instructions}}) {
  console.log('name:', name);
```

It's roughly the same but we've found it to be a convenient pattern. Otherwise you will sometimes get a mix of people using `props.chore.name` later in the file and also `chore.name`.